### DISCUSSION TITLE
Reduce memory usage for machine learning module

### DISCUSSION THREAD

**Discussion by DanielAndreasen** [immich-app/immich#3559] [Q&A]
I only have 4gb available on my homeserver, and with other services running I am a bit limited. I see that the machine learning module take up around 1gb.
Is there a way to reduce this, or maybe even turn it off?

I use docker-compose to run all the services.

**Comment by bo0tzz** [comment]
You can disable ML: https://immich.app/docs/FAQ#how-to-disable-machine-learning-and-typesense
